source/one-or-more.d.ts

```diff
@@ -0,0 +1,22 @@
+/**
+Either matches an array containing only `Value`, or `Value` itself, not in an array.
+
+This can prove to be useful when you're not sure if you're going to get a type as an array, or on it's own.
+
```

Comment:
Any TypeScript issue tracker discussion about this type or something similar?

Reply:
Since you asked, I'm guessing you do?

Reply:
I don't, but it's the submitters task to search and try to find one, per the contribution guidelines.